java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -1028,4 +1036,81 @@ public void testRemoveSpecificAPIpatterns() throws IOException {
             "}")
         .doTest();
   }
+
+  public void transformAndCreateNewPropertyFile(
+      String srcProp, String trgtProp, String stale_flag, boolean isTreated) throws IOException {
+
+    String[] temp = stale_flag.split("_");
+    String flagNameCamelCase =
+        temp[0]
+            + Arrays.stream(temp, 1, temp.length)
+                .map(str -> str.substring(0, 1).toUpperCase() + str.substring(1))
+                .collect(joining());
+    String newContent =
+        Files.readAllLines(Paths.get(srcProp))
+            .stream()
+            .map(x -> x.replace("$flagName$", flagNameCamelCase))
+            .map(x -> x.replace("$notIsTreated$", String.valueOf(!isTreated)))
```

Comment:
any reason for representing this as a negation instead of `isTreated`?

Reply:
Yes. This is how we can ensure that the method associated to the @BoolParm annotation is always deleted.